app/src/main/java/nextstep/jwp/dashboard/controller/dto/UserDto.java

```diff
@@ -1,34 +1,47 @@
-package nextstep.jwp.model;
+package nextstep.jwp.dashboard.controller.dto;
 
-public class User {
+import nextstep.jwp.dashboard.domain.User;
 
-    private final long id;
+public class UserDto {
+    private final Long id;
     private final String account;
     private final String password;
     private final String email;
 
-    public User(long id, String account, String password, String email) {
+    public UserDto(User user) {
+        this(user.getId(), user.getAccount(), user.getPassword(), user.getEmail());
+    }
+
+    public UserDto(Long id, String account, String password, String email) {
```

Comment:
확인해보니 이 생성자를 사용하는 곳이 한 곳뿐이어서
생성자를 private으로 두고 정적팩토리 메서드를 사용할 수도 있겠네요!
반영은 선택입니당 ㅎㅎ

```suggestion
    public static UserDto from(User user) {
        new User(user.getId(), user.getAccount(), user.getPassword(), user.getEmail());
    }

    private UserDto(Long id, String account, String password, String email) {
```